install.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/sh -eu
+
+#
+# Installer.
+#
+
+INSTALL_ROOT=""
+MACHINE="nicole"
```

Comment:
for compatibility with other platforms this value can be read from `/etc/os-release`

Reply:
This script is only for installation purpose, it is not used in runtime.